v2/clients/http/system.go

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright (C) 2021 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package http
+
+import (
+	"context"
+	"path"
+	"strings"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/errors"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/clients/http/utils"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/clients/interfaces"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/dtos/common"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/dtos/requests"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/dtos/responses"
+)
+
+type SystemManagementClient struct {
+	baseUrl string
+}
+
+func NewSystemManagementClient(baseUrl string) interfaces.SystemManagementClient {
+	return &SystemManagementClient{
+		baseUrl: baseUrl,
+	}
+}
+
+func (smc *SystemManagementClient) GetHealth(ctx context.Context, services []string) (res responses.HealthResponse, err errors.EdgeX) {
+	urlPath := path.Join(v2.ApiBase, "health", strings.Join(services, v2.CommaSeparator))
```

Comment:
These need to change to using query parameter for services list per Core WG discussion on 5/27/21